invenio_rdm_records/services/services.py

```diff
@@ -295,3 +295,15 @@ def delete_secret_link(
         self._index_related_records(record, parent)
 
         return True
+
+    def reseve_doi(
+        self,
+        id_,
+        doi,
+
+    )
+
+
+    def unreserve_doi(
```

Comment:
moderate: You are calling it above with the following parameters,

```
self.service.unreserve_doi(
             id_=resource_requestctx.view_args["pid_value"],
             identity=g.identity,
             doi=resource_requestctx.view_args["doi"],
         )
```

However, they are missing here.